README.md

```diff
@@ -4,34 +4,41 @@ Bundle for interfacing UFS models with JEDI interfaces
+
+  1) intel-oneapi-compilers/2021.3.0  27) gettext/0.21      53) libtirpc/1.2.6               79) py-et-xmlfile/1.0.1
+  2) stack-intel/2021.3.0             28) libunistring/1.1  54) hdf/4.2.15                   80) py-jdcal/1.3
+  3) intel-oneapi-mpi/2021.3.0        29) libidn2/2.3.4     55) jedi-cmake/1.4.0             81) py-openpyxl/3.0.7
+  4) stack-intel-oneapi-mpi/2021.3.0  30) pcre2/10.42       56) libxt/1.1.5                  82) py-six/1.16.0
+  5) bacio/2.4.1                      31) git/2.40.0        57) libxmu/1.1.4                 83) py-python-dateutil/2.8.2
+  6) snappy/1.1.10                    32) nccmp/1.9.0.1     58) libxpm/3.5.12                84) py-pytz/2023.3
+  7) zlib/1.2.13                      33) py-pip/23.1.2     59) libxaw/1.0.13                85) py-pandas/1.5.3
+  8) zstd/1.5.2                       34) wget/1.21.2       60) udunits/2.2.28               86) py-pybind11/2.8.1
+  9) c-blosc/1.21.4                   35) base-env/1.0.0    61) ncview/2.1.8                 87) py-pycodestyle/2.8.0
+ 10) curl/8.1.2                       36) boost/1.78.0      62) netcdf-cxx4/4.3.1            88) py-pyhdf/0.10.4
+ 11) pkg-config/0.29.2                37) bufr/12.0.1       63) json/3.10.5                  89) libyaml/0.2.5
+ 12) hdf5/1.14.0                      38) ecbuild/3.7.2     64) json-schema-validator/2.1.0  90) py-pyyaml/5.4.1
+ 13) netcdf-c/4.9.2                   39) openjpeg/2.3.1    65) odc/1.4.6                    91) py-packaging/23.1
+ 14) netcdf-fortran/4.6.0             40) eccodes/2.27.0    66) py-attrs/21.4.0              92) py-xarray/2022.3.0
+ 15) parallel-netcdf/1.12.2           41) eigen/3.4.0       67) py-pycparser/2.21            93) jedi-base-env/1.0.0
+ 16) parallelio/2.5.10                42) openblas/0.3.19   68) py-cffi/1.15.1               94) gftl/1.10.0
+ 17) esmf/8.5.0                       43) eckit/1.24.4      69) py-findlibs/0.0.2            95) gftl-shared/1.6.1
+ 18) fms/2023.02.01                   44) fftw/3.3.10       70) py-setuptools/59.4.0         96) fargparse/1.5.0
+ 19) libjpeg/2.1.0                    45) fckit/0.11.0      71) py-numpy/1.22.3              97) mapl/2.40.3-esmf-8.5.0
+ 20) jasper/2.0.32                    46) fiat/1.2.0        72) py-eccodes/1.4.2             98) w3emc/2.10.0
+ 21) libpng/1.6.37                    47) ectrans/1.2.0     73) py-f90nml/1.4.3              99) nemsio/2.5.4
+ 22) g2/3.4.5                         48) atlas/0.35.0      74) py-h5py/3.7.0               100) sfcio/1.4.1
+ 23) g2tmpl/1.10.2                    49) git-lfs/3.0.2     75) py-cftime/1.0.3.4           101) sigio/2.3.2
+ 24) sp/2.3.3                         50) gsibec/1.1.3      76) py-netcdf4/1.5.8            102) w3nco/2.4.1
```

Comment:
This is reverting my recent change in `README.md` that replaced `w3nco` with `w3emc`

Reply:
Ah, I missed that. I mostly wanted to update the fms version to 2023.02.01 which is now required by the WM. I'll get that straightened out.